tui_main.c

```diff
@@ -1228,8 +1230,6 @@ void ged_adjust_bpm(Ged* a, Isz delta_bpm) {
   Isz new_bpm = (Isz)a->bpm + delta_bpm;
   if (new_bpm < 1)
     new_bpm = 1;
-  else if (new_bpm > 3000)
-    new_bpm = 3000;
```

Comment:
I also removed the upper bounds for changing the bpm at runtime, as setting a higher bpm at startup and then incrementing it would result in it dropping down to 3000.
An alternative solution would have been to add an upper bound to the command-line option, which I would find too restricting (making it easier to use Orca-c for DSP was a big motivator for this PR).